dev-tools/package_test.go

```diff
@@ -0,0 +1,289 @@
+package dev_tools
+
+// This file contains tests that can be run on the generated packages.
+// To run these tests use `go test package_test.go`.
+
+import (
+	"archive/tar"
+	"archive/zip"
+	"bytes"
+	"compress/gzip"
+	"flag"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/blakesmith/ar"
```

Comment:
Yeah, I can move these to dev-tools. I was thinking about that when I added them because `glide` was giving lots of errors and also because I wanted it to be clear that these deps were not used in any Beat that we release.